### PR TITLE
Removed participant_validation feature flag

### PR DESCRIPTION
**Pull request by BroiSatse**
Removing :participant_validation feature flag!

### CODE REVIEW

File: app/services/validation_beta_service.rb [hunk 1]
```diff
@@ -1,46 +1,6 @@
...
-
-  def tell_induction_coordinators_to_check_ect_and_mentor_information
-    InductionCoordinatorProfile.find_each do |ic|
-      if ic.schools.not_opted_out.any? { |school| chosen_programme_and_not_in_beta(school) }
-        send_check_ect_and_mentor(ic)
-      end
-    end
-  end
-
-  def tell_induction_coordinators_we_asked_ects_and_mentors_for_information
```
Comment (jamesrose) [inline app/services/validation_beta_service.rb:33]: We need this one - it's #10 on the schedule, we haven't sent it yet.

File: config/routes.rb [hunk 2]
```diff
@@ -254,7 +254,7 @@
     resource :no_access, only: :show, controller: "no_access"
     resource :start_registrations, path: "/start-registration", only: :show
 
-    authenticated :user, ->(user) { FeatureFlag.active?(:participant_validation, for: user.teacher_profile&.participant_profiles&.ecf&.active_record&.first&.school) } do
+    authenticated :user do
```
Comment (tonyheadford) [inline config/routes.rb:257]: Think this can go as it was only needed to get the context for the feature flag check

**Review by jamesrose — APPROVED** [review]
Nice one 🚢

File: config/routes.rb (hunk 2 above)
Comment (BroiSatse) [reply config/routes.rb:257]: Ah nice. Also, we have a multistep form for this kind of thing, I'll try to change it later on. :)

**Pull request merged**